tests/001.phpt

```diff
@@ -43,6 +43,7 @@ AST_STMT_LIST
                 0: AST_IF_ELEM
                     cond: AST_INSTANCEOF
                         expr: AST_VAR
+                            flags: 0
```

Comment:
Hm, I think we might want to adjust the printer to not print a 0 flag value for combinable flags.

Reply:
That sounds fine to me since I expect this to be meant more for human readable debugging output (and util.php would not be installed in composer; this is a pecl)

There'll still be some `flags: 0\n` lines after this, because some flags are mutually exclusive (e.g. absence of a type for AST_GROUP_USE is 0, not USE_NORMAL)

```c
const USE_NORMAL = 1;
const USE_FUNCTION = 2;
const USE_CONST = 4;
```